chainer/functions/clipped_relu.py

```diff
@@ -0,0 +1,69 @@
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+import numpy
+
+
+class ClippedReLU(function.Function):
+
+    """clipped ReLU function. clipped ReLU is written as below,
```

Comment:
Please write a brief explanation of the class in the first line, and write the details after that.